tests/admin_docs/test_views.py

```diff
@@ -293,6 +294,25 @@ def test_model_docstring_renders_correctly(self):
     def test_model_detail_title(self):
         self.assertContains(self.response, '<h1>admin_docs.Person</h1>', html=True)
 
+    def test_app_not_found(self):
+        response = self.client.get(
+            reverse('django-admindocs-models-detail', args=['doesnotexist', 'Person'])
+        )
+        self.assertEqual(response.context['exception'], "App 'doesnotexist' not found")
+        self.assertEqual(response.status_code, 404)
+        self.assertIsInstance(response, HttpResponseNotFound)
```

Comment:
I think checking `response.status_code` is enough, did you have any specific thinking about verifying the class?

Reply:
Not really. The instance check is also done in addition to the status code in a few other tests, e.g., `test_response_redirect_class_subclass`. I removed the instance check.